tests/modules/icgc-argo-workflows/song/publish/test.yml

```diff
@@ -1,6 +1,6 @@
 
 - name: "song publish"
-  command: nextflow run ./tests/modules/icgc-argo-workflows/song/publish -entry test_song_publish -c ./tests/config/nextflow.config -profile docker,debug_qa --api_token YOUR_TOKEN
+  command: nextflow run ./tests/modules/icgc-argo-workflows/song/publish -entry test_song_publish -c ./tests/config/nextflow.config -profile docker,rdpc_qa --api_token YOUR_TOKEN
```

Comment:
updated to `rdpc_qa` in song/publish. Test run with error. 

In `test_data.config` `analysis_id_published = "e3ae259f-753b-402b-ae25-9f753be02b23"` need to be updated with analysis with latest version. 

As this is test is prepared by @edsu7, could you please help update the analysis_id